tests/db/conftest.py

```diff
@@ -30,23 +30,22 @@ def setup_database(request, node_config):
     db_name = node_config['database']['name']
     conn = connect()
 
-    if conn.run(r.db_list().contains(db_name)):
-        conn.run(r.db_drop(db_name))
-
     try:
-        schema.init_database()
+        schema.init_database(conn)
     except DatabaseAlreadyExists:
         print('Database already exists.')
+        schema.drop_database(conn, db_name)
+        schema.init_database(conn)
```

Comment:
Might be nice to restructure this to avoid duplicating the `init_database` call:

```python
try:
    schema.drop_database(conn, db_name)
except DatabaseDoesNotExist:
    pass
schema.init_database(conn)
```